__tests__/app.test.js

```diff
@@ -41,19 +41,50 @@ describe("API Testing", () => {
 	});
 	describe("Articles Endpoint", () => {
 		test("Get: 200, should return a single article", () => {
-			return request(app).get("/api/articles/1").expect(200);
+			return request(app)
+				.get("/api/articles/3")
+				.expect(200)
+				.then(({ body }) => {
+					const { article } = body;
+					expect(article).toBeInstanceOf(Array);
+					expect(article.length).not.toBeLessThan(1);
+					article.forEach((article) => {
+						expect(article).toHaveProperty("author");
+						expect(article).toHaveProperty("title");
+						expect(article).toHaveProperty("article_id");
```

Comment:
You should be more specific here, which article id are you expecting this to be?